setup.cfg

```diff
@@ -29,9 +29,10 @@ setup_requires =
     setuptools_scm_git_archive
 install_requires =
     click
-    tabulate
-    python-gitlab
+    ogr
     pygithub
+    python-gitlab
```

Comment:
Those two are used only via ogr, aren't they?

(Or at least, everything should be done via ogr and we need to create issues in OGR for missing pieces.)

Reply:
No, they are not. Both of them are imported in the current `upsint` code.

Reply:
OK, so we need to fix that (out-of-scope of this PR) or create issues.